pages/Batches/FileFormat.md

```diff
@@ -14,42 +15,57 @@ Batch files are stored and transferred in Apache Parquet format.
 
 Note this carries implications for row group size as well as configuration of Bloom filters for the batch file.
 
-## Bloom filter, row group size 
-Optimal settings are still under investigation, however, the maximum row group size allowed in a Parquet file (128*1024*1024 rows) is far too large for a browser or small client application to handle in JavaScript. We are currently defaulting to 128*1024 rows.  
+## Bloom filter, row group size
+
+Optimal settings are still under investigation, however, the maximum row group size allowed in a Parquet file (128*1024*1024 rows) is far too large for a browser or small client application to handle in JavaScript.
+We are currently defaulting to 128*1024 rows.  
 
-In Parquet, the Bloom filter type is Split Block; the calculation for filter bits is different and nearly a factor of 10 lower than for the normal Bloom filter.  128*1024 rows with a 0.001 false positive rate results in around 29,000 bits for a Split Block Bloom filter.
+In Parquet, the Bloom filter type is Split Block; the calculation for filter bits is different and nearly a factor of 10 lower than for the normal Bloom filter.
+128*1024 rows with a 0.001 false positive rate results in around 29,000 bits for a Split Block Bloom filter.
 
 Absent benchmarks, the False Positive Rate currently defaults to 0.001.
- 
+
 ## API
-Batch announcements are done via the SDK API for Announcements.  Bloom filter settings are set as described above in the SDK.  This may be configurable without changing the source code at a later date.  
+
+Batch announcements are done via the SDK API for Announcements.
+Bloom filter settings are set as described above in the SDK.
+This may be configurable without changing the source code at a later date.  
 
 ## Benchmarks
+
 TBD
 
 ## Justifications
 
 ### Requirements
-Batch files need to be quickly and easily searchable. Minimal storage size and fast, simple querying, identification and transfer are preferred to guarantees of no false positives or advanced data manipulation and column relationships.  The files ideally are parseable by client applications, web views or browsers running pure JavaScript without a need to convert the format.
 
-DSNP doesn't need a database; the data is not relational. The columns:rows ratio are  at most 1:1000, likely more at 1:10,000, within one batch file. Applications need to know if a given Batch file has any information they are interested in without downloading the file first.
+Batch files need to be quickly and easily searchable.
+Minimal storage size and fast, simple querying, identification and transfer are preferred to guarantees of no false positives or advanced data manipulation and column relationships.
```

Comment:
This sentence is is grammatically correct if "fast, simple" apply to all of "querying", "identification" and "transfer", but it's difficult to parse. Suggest dropping "identification and transfer" because those are subsumed under the normal definition of "querying".

Reply:
Dropped